### PR TITLE
Serilog log to zip file

### PR DESCRIPTION
**Pull request by Mbucari**
This is really cool. I created a new sink for serilog, [Serilog.Sinks.ZipFile](https://github.com/Mbucari/serilog-sinks-zipfile/tree/master/src/Serilog.Sinks.ZipFile), to directly into a zip file.
From the outside it behaves exactly like `Serilog.Sinks.File`. Naming, filename increments, rolling intervals, etc. The difference are:

- The log text is created as a file inside the zip file (with no compression to improve logging performance)
- Supplemental files can be added [as described in the docs](https://github.com/Mbucari/serilog-sinks-zipfile#logging-files-into-the-zip)

The zip is saved after each log event is written.

### CODE REVIEW

File: Source/LibationAvalonia/Dialogs/Login/LoginChoiceEagerDialog.axaml [hunk 1]
```diff
@@ -60,7 +60,7 @@
 
 			<controls:LinkLabel
 				Tapped="ExternalLoginLink_Tapped"
-				Text="Or click here to log in with your browser." />
+				Text="Trouble logging in? Click here to log in with your browser." />
```
Comment (rmcrackan) [inline Source/LibationAvalonia/Dialogs/Login/LoginChoiceEagerDialog.axaml:63]: I've been calling it the 'or click here' link for so long. I wonder how many times I'll call this the wrong thing

**Comment by rmcrackan** [comment]
Do you want me to approve the api pr, then update refs, and finally update libation pr?

**Comment by rmcrackan** [comment]
nuget `AudibleApi 8.3.0.1` is available. Please include that upgrade in this PR

**Comment by Mbucari** [comment]
Yes please. Also, I don't know if you want to keep the [Serilog.Sinks.ZipFile](https://github.com/Mbucari/serilog-sinks-zipfile/tree/master/src/Serilog.Sinks.ZipFile) reference in AppScaffolding or if you want to add it to Dinah.Core (replacing Serilog.Sinks.File).

**Comment by Mbucari** [comment]
> nuget `AudibleApi 8.3.0.1` is available. Please include that upgrade in this PR

Alright I will lol

**Comment by rmcrackan** [comment]
> keep the Serilog.Sinks.ZipFile reference in AppScaffolding or if you want to add it to Dinah.Core (replacing Serilog.Sinks.File)

No strong feelings. Do you have a preference?

**Comment by Mbucari** [comment]
> No strong feelings. Do you have a preference?

None whatsoever.

I also added a check to fix #585

**Comment by rmcrackan** [comment]
Good catch. Any other changes before I merge? Also -- new release today? Or wait until there's more substance added?

**Comment by Mbucari** [comment]
> Any other changes before I merge

Nope, I'm done.

> new release today? Or wait until there's more substance added?

I think make a new release. I don't have anything else in the works right now, and I probably won't until after my vacation from Apr 28 - May 7. Between now and then I've got a lot of other things to catch up on, so I think I'll only be doing bug fixes until after I return from vacation.

**Comment by rmcrackan** [comment]
New version released

**Pull request merged**